### PR TITLE
[NET 6 Attribute Conversion] CFNetwork

### PR DESCRIPTION
**Pull request by chamons**

### CODE REVIEW

**Review by stephen-hawley — APPROVED** [review]

File: src/CFNetwork/CFHost.cs [hunk 1]
```diff
@@ -28,26 +28,26 @@ namespace CFNetwork {
...
-#elif TVOS
-	[Obsolete ("Starting with tvos15.0 use 'Network.framework' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+	[UnsupportedOSPlatform ("ios15.0")]
+#if TVOS
+	[Obsolete ("Starting with tvos15.0 onstants.UseNetworkInstea.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#elif __MACCATALYST__
+	[Obsolete ("Starting with maccatalyst15.0 onstants.UseNetworkInstea.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
 #elif MONOMAC
-	[Obsolete ("Starting with macos12.0 use 'Network.framework' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+	[Obsolete ("Starting with macos12.0 onstants.UseNetworkInstea.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
```
Comment (rolfbjarne) [inline src/CFNetwork/CFHost.cs:41]: This consant isn't quite right...

**Review by rolfbjarne — CHANGES_REQUESTED** [review]

File: src/CFNetwork/CFHost.cs (hunk 1 above)
Comment (chamons) [reply src/CFNetwork/CFHost.cs:41]: Let me look into what the heck mellite is smoking...

**Review by dalexsoto — APPROVED** [review]
Other than Rolf's findings, it looks good to me

File: src/CFNetwork/CFHost.cs (hunk 1 above)
Comment (chamons) [reply src/CFNetwork/CFHost.cs:41]: Oh, it's a variable, and mellite just sees "Constants.UseNetworkInstead". Talk about a case I haven't seen before.

Comment (chamons) [reply src/CFNetwork/CFHost.cs]: I have corrected this by teaching the tool to look for constants and spitting out a build breaking [Verify], which needs manual fixing.

Comment (chamons) [reply src/CFNetwork/CFHost.cs]: https://github.com/chamons/mellite/commit/226eb1f9ede3e6f152a194b3c1fef52910f620e6

**Review by mandel-macaque — APPROVED** [review]
Yet rolf comment has to be fixed.

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**